packages/shared/src/lib/auth.ts

```diff
@@ -36,7 +36,6 @@ export enum AuthTriggers {
   LegacyLogout = 'legacy_logout',
   Verification = 'verification',
   VerifySession = 'verify session',
-  CreateFeedFilters = 'create feed filters',
```

Comment:
Is not used? So what's the trigger of the onboarding?

Reply:
Looks to be non tracked anymore from onboarding as it's not invoking "showLogin" but a fixed version.
It does have some other tracking events inside of it.